src/middlewares/middlewares.ts

```diff
@@ -0,0 +1,20 @@
+import jwt from 'jsonwebtoken'

```

Comment:
W folderze middlewar powinny się znaleźć middleware'y per plik. W innym wypadku jakby pojawiło się ich więcej to plik będzie co raz mniej czytelny